test/boost/cql_query_test.cc

```diff
@@ -153,6 +153,74 @@ SEASTAR_TEST_CASE(test_twcs_max_window) {
     });
 }
 
+SEASTAR_TEST_CASE(test_twcs_restrictions_mixed) {
+    return do_with_cql_env_thread([](cql_test_env& e) {
+        // Cenario 1: STCS->TWCS with no TTL defined
+        e.execute_cql("CREATE TABLE tbl (a int PRIMARY KEY, b int);").get();
+        e.execute_cql("ALTER TABLE tbl WITH compaction = {'class': 'TimeWindowCompactionStrategy'};").get();
+        e.require_table_exists("ks", "tbl").get();
+
+        // Cenario 2: STCS->TWCS with small TTL
+        e.execute_cql("CREATE TABLE tbl2 (a int PRIMARY KEY, b int) WITH default_time_to_live = 60;").get();
+        e.execute_cql("ALTER TABLE tbl2 WITH compaction = {'class': 'TimeWindowCompactionStrategy'};").get();
```

Comment:
Hmm, do we have a default TWCS window size? I didn't realize this. What is it?

Reply:
Yes, we do. 1 day.

```cpp
    std::chrono::seconds sstable_window_size = DEFAULT_COMPACTION_WINDOW_UNIT * DEFAULT_COMPACTION_WINDOW_SIZE;
```

and later on:
```cpp
    static constexpr std::chrono::seconds DEFAULT_COMPACTION_WINDOW_UNIT = 86400s;
    static constexpr int DEFAULT_COMPACTION_WINDOW_SIZE = 1;
```